catroid/src/main/java/org/catrobat/catroid/common/Constants.java

```diff
@@ -282,6 +282,10 @@
 	public static final String ZIP_FILE_NAME = START_PROJECT + ".zip";
 	public static final String STANDALONE_PROJECT_NAME = BuildConfig.PROJECT_NAME;
 
+	//Restricted login
+	public static final String RESTRICTED_LOGIN_ACCEPTED = "restrictedLoginAccepted";
+	public static final String RESTRICTED_LOGIN_LINK_GERMAN = "http://catrob.at/schulregistrierung";
```

Comment:
why not move this to strings xml?

Reply:
the link is only availble in Englisch and German and it should not be translated via crowdin